lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3371,6 +3371,11 @@ namespace {
           }
         }
 
+        // If we've already imported this decl, skip it so we don't add the same
+        // member twice.
+        if (Impl.ImportedDecls.count({nd->getCanonicalDecl(), getVersion()}))
```

Comment:
Nit: I think it would be more idiomatic to use `find() != end()` instead of `count`, until someone adds a `contains_key` method to upstream DenseMap.

Reply:
Will fix, thanks!